src/pages/ProcessJoinTeam.vue

```diff
@@ -93,24 +103,28 @@ export default {
 		preFetchVariables({ route }) {
 			return {
 				teamPublicId: route.query.teamPublicId,
+				publicId: route.query.inviter ?? '',
```

Comment:
If the inviter query param is not present, this just sends an empty string as the param, the api just returns lender null and the page functions fine